zkevm-circuits/src/evm_circuit/util.rs

```diff
@@ -84,20 +86,52 @@ impl<'r, 'b, F: FieldExt> CachedRegion<'r, 'b, F> {
     pub(crate) fn new(
         region: &'r mut Region<'b, F>,
         power_of_randomness: [F; 31],
-        width: usize,
+        advice_columns: Vec<Column<Advice>>,
         height: usize,
-        width_start: usize,
         height_start: usize,
     ) -> Self {
         Self {
             region,
-            advice: vec![vec![F::zero(); height]; width],
+            advice: vec![vec![F::zero(); height]; advice_columns.len()],
             power_of_randomness,
-            width_start,
+            width_start: advice_columns[0].index(),
             height_start,
+            advice_columns,
         }
     }
 
+    /// Repeatedly assign the locally cached witnesses.
+    /// This method can be used as a "quick" path for assignment for repeated
+    /// padding rows
+    pub fn replicate_assignment_for_range<A, AR>(
```

Comment:
I would suggest the following small change to this function comment in order to make it more clear what it does:

Add to the comment indicating that this function replicates the assignment of 1 row at `height_start` (which must be already assigned via the `CachedRegion`) into a range of rows indicated by `offset_begin, offset_end`.

Reply:
done